src/interpret.jl

```diff
@@ -538,14 +538,18 @@ function step_expr!(@nospecialize(recurse), frame, @nospecialize(node), istoplev
                     error("this should have been handled by split_expressions")
                 elseif node.head === :using || node.head === :import || node.head === :export
                     Core.eval(moduleof(frame), node)
-                elseif node.head === :const
+                elseif node.head === :const || node.head === :globaldecl
                     g = node.args[1]
                     if isa(g, GlobalRef)
                         mod, name = g.mod, g.name
                     else
                         mod, name = moduleof(frame), g::Symbol
```

Comment:
Looks like now we don't need this `mod` variable? Assuming these `GlobalRef` modules are always same as `moduleof(frame)`.

Reply:
We just just remove this block and use `g` directly, since we're falling back to the base interpreter anyway.